--- libdvdread-0.9.3/dvdread/dvd_reader.c~	Sun May 19 17:48:41 2002
+++ libdvdread-0.9.3/dvdread/dvd_reader.c	Mon Jun 24 18:48:31 2002
@@ -769,7 +769,12 @@
 		off = DVDinput_seek( dvd_file->title_devs[ i ], 
 				     (int)offset, DVDINPUT_NOFLAGS );
                 if( off < 0 || off != (int)offset ) {
-		    fprintf( stderr, "libdvdread: Can't seek to block %d\n", 
+		    fprintf( stderr, "libdvdread: Can't seek to block "
+#ifdef __alpha__
+				    "%ld\n",
+#else
+				    "%d\n",
+#endif
 			     offset );
 		    return off < 0 ? off : 0;
 		}
@@ -785,7 +790,12 @@
                 off = DVDinput_seek( dvd_file->title_devs[ i ], 
 				     (int)offset, DVDINPUT_NOFLAGS );
                 if( off < 0 || off != (int)offset ) {
-		    fprintf( stderr, "libdvdread: Can't seek to block %d\n", 
+		    fprintf( stderr, "libdvdread: Can't seek to block "
+#ifdef __alpha__
+				    "%ld\n",
+#else
+				    "%d\n",
+#endif
 			     offset );
 		    return off < 0 ? off : 0;
 		}
