--- libdvdread-4.2.0/src/dvd_input.c.orig	2009-06-09 00:02:37.000000000 +0200
+++ libdvdread-4.2.0/src/dvd_input.c	2012-03-16 20:01:21.501749182 +0100
@@ -218,12 +218,13 @@ static int file_read(dvd_input_t dev, vo
 {
   size_t len;
   ssize_t ret;
+  unsigned char *buf = buffer;
 
   len = (size_t)blocks * DVD_VIDEO_LB_LEN;
 
   while(len > 0) {
 
-    ret = read(dev->fd, buffer, len);
+    ret = read(dev->fd, buf, len);
 
     if(ret < 0) {
       /* One of the reads failed, too bad.  We won't even bother
@@ -242,6 +243,7 @@ static int file_read(dvd_input_t dev, vo
       return (int) (bytes / DVD_VIDEO_LB_LEN);
     }
 
+    buf += ret;
     len -= ret;
   }
 
